packages/core/components/IconButton/IconButton.module.css

```diff
@@ -3,7 +3,7 @@
   background: transparent;
   border: none;
   border-radius: 4px;
-  color: var(--puck-color-grey-3);
+  color: currentColor;
```

Comment:
This unexpectedly causes the icon colour to change in some scenarios (such as the array item action overlay). Curious to know the reason?

Fix might be to change the color of the array item action overlay wrapper.

Reply:
I wanted the sidebar toggle icons to have a different a colour depending if the sidebar is open or not, this was not possible with a hardcoded colour for the `IconButton`, so switched to inherit from parent. In general inherit is a more composable pattern.

Will update array field action 👍🏻

Reply:
Done